pages/pages.json

```diff
@@ -0,0 +1,6 @@
+{
+  "index": [],
+  "explore-products": [],
+  "phone-numbers": ["active-numbers"],
+  "template-page": []
+}
```

Comment:
Maybe you're already planning on doing this, but it might be helpful to have in this file some commented lines explaining how to add pages/nested pages?

Reply:
I see that `components/site/Sidebar/SideNavList.js` renames `index` to Home when it renders, but do you think it might cause an issue for a user that wants to rename the Home page (or cause confusion about why it says 'index' here and 'Home' on the prototype)? Is there a reason that change happens in `SideNavList.js` rather than all the keys in the `pages` object being named accurately?

Reply:
Is the assumption here that the page filename matches these exactly?

Reply:
- Potential solution for commenting: JSON does not support commenting. However, our solution to this could be to have a key-value pair of instructions in the pages.json file that SideNavList ignores based on a conditional statement (similar to what it does to index right now). Could look like:

{
  "HOW TO CREATE YOUR NAVIGATION" : "Populate your side navigation bar by adding a key-value pair to this file inside of the curly braces.",
  "home" : [],
}

We would work on the wording...

- @nkrantz for your second question: that renaming was for when we dynamically looked at the pages directory. Since we don't do that anymore, I think you're right and we could just call the first link home at the pages.json stage, and replace the renaming conditional with 'home' not index. Thanks for catching this!

- @SiTaggart we are assuming the page filename matches these exactly, since that's the name that's used to create the routes that the links refer to.

Reply:
> we are assuming the page filename matches these exactly, since that's the name that's used to create the routes that the links refer to.

Perfect. Something we can document in the getting started docs for the prototyping kit

Reply:
And yeah, as JSON doesn't support comments, I wouldn't necessarily add a key for instructions here. I would just document it really well

Reply:
@SiTaggart re: documenting nav editing without adding another key in the json: sounds good. I've added instructions for it in the landing page.

Also replaced index in the pages.json file and in the filtering in SideNavList with "home" for simplicity.